### ISSUE TITLE
Should we join as a neat-python team ? :)

### ISSUE THREAD

**Issue by d0pa** [CodeReclaimers/neat-python#64]
In case anyone interested  to join as individual or a team can register for General AI challenge. 
https://www.general-ai-challenge.org/

Bonus free Microsoft Azure cloud space with 16 cores or 6 cores + GPU

**Comment by CodeReclaimers** [comment]
Hey sorry I didn't respond earlier, been overwhelmed with work. :P  That might be an interesting thing to work on; although I know I don't know nearly enough to be good at it, it would be nice to beat up neat-python in a project like that and make it easier to use in a multi-core/GPU environment (especially if Microsoft is providing cloud computing resources for free).

**Comment by d0pa** [comment]
Great you are back ! I looked at there problems example more closely some should be very easy to solve with neat. I finished today implementation of population.py with ipyparallel/mpi4py, where data is scattered across nodes/processes. About to test it on google compute engine (since is free $300) for each subscription. Next version will do where genomes are scattered across nodes processes. I am not sure about GPU, because don't have clear picture where GPU will improve execution speed yet. 
Challenge is interesting after asking questions I am not impressed with them :(, although it will be good for project visibility and for neat-python development. So far it looks to me that neat-python can be competitive in Qualitative prize. I sent you email, looking forward to registering a team !
